Sources/Commands/SwiftPackageTool.swift

```diff
@@ -259,6 +262,7 @@ public struct SwiftPackageTool: SwiftTool {
         print("OPTIONS:")
         print("  -C, --chdir <path>        Change working directory before any other operation")
         print("  --color <mode>            Specify color mode (auto|always|never)")
+        print("  --enable-code-coverage    Enable code coverage in generatd Xcode projects")
```

Comment:
That's fine (not mentioning test).  I don't have a strong opinion, it just seemed like something to explain.  In a man page version or hypothetical `--long-help` option it would be something to go into.